### PR TITLE
[GR] Bump GenStage to latest version

### PR DESCRIPTION
**Pull request by RTSolo**

### CODE REVIEW

File: gr/lessons/advanced/gen-stage.md [hunk 1]
```diff
@@ -84,27 +95,33 @@ $ touch lib/genstage_example/producer.ex
 ```elixir
 defmodule GenstageExample.Producer do
   use GenStage
-
+
```
Comment (gemantzu) [inline gr/lessons/advanced/gen-stage.md:98]: please remove the empty space (tabs?) here.

File: gr/lessons/advanced/gen-stage.md [hunk 2]
```diff
@@ -115,34 +132,39 @@ $ touch lib/genstage_example/producer_consumer.ex
 ```elixir
 defmodule GenstageExample.ProducerConsumer do
   use GenStage
-
+
```
Comment (gemantzu) [inline gr/lessons/advanced/gen-stage.md:135]: Be careful of your empty spaces.

File: gr/lessons/advanced/gen-stage.md [hunk 3]
```diff
@@ -11,39 +11,49 @@ title: GenStage
...
 
 ## Καταναλωτές και Παραγωγοί
 
-Όπως έχουμε διαβάσει, ο ρόλος που δίνουμε στο στάδιό μας είναι σημαντικός. Οι προδιαγραφές του GenStage αναγνωρίζουν τρεις ρόλους:
+Όπως έχουμε διαβάσει, ο ρόλος που δίνουμε στο στάδιό μας είναι σημαντικός.
+Οι προδιαγραφές του GenStage αναγνωρίζουν τρεις ρόλους:
 
-+ `:producer` — Μια πηγή.  Οι παραγωγοί περιμένουν για ζήτηση από τους καταναλωτές και ανταποκρίνονται με τα συμβάντα που ζητήθηκαν.
++ `:producer` — Μια πηγή.
+Οι παραγωγοί περιμένουν για ζήτηση από τους καταναλωτές και ανταποκρίνονται με τα συμβάντα που ζητήθηκαν.
```
Comment (gemantzu) [inline gr/lessons/advanced/gen-stage.md:35]: I have to check if this works prorperly as, it might break this list. Let me get back to you for this one.

**Comment by RTSolo** [comment]
@gemantzu Thanks for the updates. 2/3 resolved. Awaiting comments on the last part.

**Review by gemantzu — APPROVED** [review]
It works perfectly fine, so leave it like that. Thanks @RTSolo!

**Pull request merged**